aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1555,19 +1555,44 @@ Routes can be [CORS](xref:security/cors?view=aspnetcore-6.0) enabled using [CORS
 
 For more information, see <xref:security/cors?view=aspnetcore-6.0>
 
+<a name="openapi7"></a>
+
 ## OpenAPI
 
 An app can describe the [OpenAPI specification](https://swagger.io/specification/) for route handlers using [Swashbuckle](https://www.nuget.org/packages/Swashbuckle.AspNetCore/).
 
-The following code is a typical ASP.NET Core app with OpenAPI support:
+The following code is generated by the ASP.NET Core minimal web API template and uses OpenAPI:
+
+[!code-csharp[](minimal-apis/7.0-samples/WebMinOpenApi/Program.cs?name=snippet&highlight=1,5-6,12-13,35-36)]
```

Comment:
We probably want to call out that the `builder.Services.AddEndpointsApiExplorer();` in the template is for the "old" ApiExplorer-based approached or endpoints here. I'd probably include something to this effect in a callout textbox after the template.

> The `builder.Services.AddEndpointsApiExplorer()` configures the application to use the ApiExplorer-based approach for generating OpenAPI annotations. Any use of `WithOpenApi` will override annotations generated by the ApiExplorer with those produced from the `Microsoft.AspNetCore.OpenApi` package.

Reply:
Fair, but we should be cautious about potentially misleading a reader. The API Explorer is still needed. API Explorer != OpenAPI. There are use cases for API exploration outside of the context of OpenAPI.

Consider a slight variation:

>The `builder.Services.AddEndpointsApiExplorer()` configures the application to use the API Explorer to discover and describe endpoints with default annotations. Any use of `WithOpenApi` will override matching, default annotations generated by the API Explorer with those produced from the `Microsoft.AspNetCore.OpenApi` package.